nomad/structs/config/ui.go

```diff
@@ -11,6 +11,9 @@ type UIConfig struct {
 	// Enabled is used to enable the web UI
 	Enabled bool `hcl:"enabled"`
 
+	// Disable frame-ancestors CSP header
+	DisableFrameAncestors bool `hcl:"disable_frame_ancestors"`
```

Comment:
We'll also need to add this field to the `(*UIConfig) Merge` method below so that configuration with multiple files works correctly.